homeassistant/components/samsungtv/media_player.py

```diff
@@ -90,91 +96,45 @@ def __init__(self, config_entry, on_script):
         # Assume that the TV is in Play mode
         self._playing = True
         self._state = None
-        self._remote = None
         # Mark the end of a shutdown command (need to wait 15 seconds before
         # sending the next command to avoid turning the TV back ON).
         self._end_of_power_off = None
         # Generate a configuration for the Samsung library
-        self._config = {
+        config = {
             "name": "HomeAssistant",
             "description": "HomeAssistant",
             "id": "ha.component.samsung",
             "method": config_entry.data[CONF_METHOD],
             "port": config_entry.data.get(CONF_PORT),
             "host": config_entry.data[CONF_HOST],
             "timeout": 1,
+            "token": config_entry.data[CONF_TOKEN],
         }
+        self._bridge = SamsungTVBridge.get_bridge(config)
+        self._bridge.register_reauth_callback(self.access_denied)
+
+    def access_denied(self):
+        """Access denied callbck."""
+        LOGGER.debug("Access denied in getting remote object")
+        self.hass.async_create_task(
```

Comment:
The callback will be called from sync context. We're not allowed to call the async api from sync context.

We can use `hass.add_job` here instead.

Reply:
I'll add it but... Just to understand: this piece of code was inside the _get_remote method. This method is (and was) not async.
Why now is a problem and it was not before? Thanks

Reply:
This same piece of code was in `_get_remote `method that AFAIK was in a sync context. 
I'll change it for sure but.... just to understand... why now is not allowed?

Reply:
It was probably wrong before too.

Reply:
Do you have an example of a `hass.add_job` call that I can use as a template?

Reply:
@MartinHjelmare this is the only part I'm missing in order to implement all your requests for change. Sorry but I need your help.
Thanks.

Reply:
Just replace `hass.async_create_task` with `hass.add_job`.

Reply:
OK, thanks.